diesel_cli/src/main.rs

```diff
@@ -286,10 +286,10 @@ fn regenerate_schema_if_file_specified(matches: &ArgMatches) -> Result<(), crate
                     // it's fine to leak here, we will
                     // exit the application anyway soon
                     let label = path.file_name().expect("We have a file name here");
-                    let label = label.to_string_lossy().into_owned().leak();
+                    let label = label.to_string_lossy();
```

Comment:
Comment above obsoleted by this.

Reply:
Thanks for pointing out :heart:. It's fixed in https://github.com/diesel-rs/diesel/pull/4269/commits/325ffc541252e40fafb367553add9ed8c9bda3cd now.